handlers/identity-retention/src/main/scala/com/gu/identityRetention/IdentityRetentionResponses.scala

```diff
@@ -33,8 +34,13 @@ object IdentityRetentionResponseModels {
       val responseValidUntil =
         if (effectiveDeletionDate isBefore today)
           today.plusMonths(3)
-        else
-          List(effectiveDeletionDate, today.plusMonths(3)).min
+        else {
+          val proportionOfMaxLifetimes = today.until(effectiveDeletionDate, ChronoUnit.DAYS) / (365.25 * 7)
```

Comment:
would be nice to put the max days in a const?  I assume the original 7 years const is not stored in this codebase.
```suggestion
          val maxLifeInDays = 365.25 * 7
          val proportionOfMaxLifetimes = today.until(effectiveDeletionDate, ChronoUnit.DAYS) / maxLifeInDays
```